src/TinyIoC/TinyIoC.csproj

```diff
@@ -92,9 +92,15 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(Configuration)'=='Source'">
-    <Content Include="TinyIoc.cs">
+    <Compile Include="TinyIoc.cs">
       <Pack>true</Pack>
       <PackageCopyToOutput>true</PackageCopyToOutput>
-    </Content>
+      <PackagePath>contentFiles/cs/any</PackagePath>
+    </Compile>
+    <Compile Include="TinyIoc.cs">
+      <Pack>true</Pack>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+      <PackagePath>content</PackagePath>
+    </Compile>

```

Comment:
I think this one was duplicated or do we need to include it twice?

Reply:
I added this to support both versions of NuGet (package.config which needs `content` and PackageReference which needs `contetnFiles/cs/any`). The second one can be dropped but then it will not work anymore on older projects.